### ISSUE TITLE
Message actions only exposed on mouse hover

### ISSUE THREAD

**Issue by MarcoZehe** [vector-im/riot-web#11082]
### Description

When reading through messages with a screen reader, the message actions to reply to a message etc., aren't visible unless you hover over the message with the mouse. Since screen reader users don't use the mouse to read, and the mouse can be placed anywhere random on the screen, but screen readers do update the DOM focused state of elements that are focusable, the message actions should be shown once a screen reader user enters a particular message.

### Steps to reproduce

In Firefox, with the NVDA screen reader:
- Enter a channel.
- Use the arrow keys and NVDA's virtual reading cursor to traverse through the messages. Watch as the screen tracks where you're reading.

Expected result: Message actions should become visible for the message the user is currently reading.

Actual: Either no message actions appear, or they only appear for the message the mouse happens to sit on, if any, and changes as the screen scrolls.

Logs being sent: yes/no

### Version information

- **Platform**: web

For the web app:

- **Browser**: Firefox 70 beta, or 71 Nightly.
- **OS**: Windows.
- **URL**: riot.im/develop

**Comment by t3chguy** [comment]
I discovered this myself yesterday and have it on my to do as part of General a11y improvements

**Comment by t3chguy** [comment]
This got regressed

**Comment by MarcoZehe** [comment]
> 
> 
> This got regressed

For book keeping, parts of the fix for this were reverted in https://github.com/matrix-org/matrix-react-sdk/pull/3591, and resulted in filing of #11316, as a possible way forward. Right now, screen readers get a toolbar that hides again from virtual cursor once focus moves into it. Not usable unless routing the mouse to the particular message, simulating a hover.
